yt_dlp/extractor/orf.py

```diff
@@ -569,19 +569,20 @@ def _real_extract(self, url):
 
 class ORFONIE(InfoExtractor):
     IE_NAME = 'orf:on'
-    _VALID_URL = r'https?://on\.orf\.at/video/(?P<id>\d{8})/(?P<slug>[\w-]+)'
+    _VALID_URL = r'https?://on\.orf\.at/video/(?P<id>\d+)(/(?P<slug>[\w-]+))?'
     _TESTS = [{
-        'url': 'https://on.orf.at/video/14210000/school-of-champions-48',
+        'url': 'https://on.orf.at/video/3220355',
```

Comment:
Don't delete the test, but add a new one.

Reply:
I can but this test case will break after 12.8.2024, 21:05
You want this?

Reply:
Yeah, keep it included for now